plugin/restore/mysql/mysql.go

```diff
@@ -31,6 +33,34 @@ const (
 	MaxDatabaseNameLength = 64
 )
 
+// BinlogFile is the metadata of the MySQL binlog file
+type BinlogFile struct {
+	Name string
+	Size int64
+
+	// Seq is parsed from Name and is for the sorting purpose.
+	Seq int64
```

Comment:
for debugging purpose, we may want to add a ts field to indicate when the file is downloaded from the server.

But it's up to you if you don't see a need for now.

Reply:
I'd like to add the field when necessary.